cmd/k8s-operator/ingress-for-pg.go

```diff
@@ -318,7 +320,7 @@ func (a *IngressPGReconciler) maybeCleanupProxyGroup(ctx context.Context, proxyG
 
 		if !found {
 			logger.Infof("VIPService %q is not owned by any Ingress, cleaning up", vipHostname)
-			svc, err := a.getVIPService(ctx, vipHostname.WithoutPrefix(), logger)
+			svc, err := a.getVIPService(ctx, vipHostname, logger)
```

Comment:
IIUC, is hostname now not quite right, because it will have a `svc:` prefix?

```suggestion
			svc, err := a.getVIPService(ctx, vipServiceName, logger)
```